internal/internal_workflow.go

```diff
@@ -52,6 +52,12 @@ const (
 	defaultCoroutineExitTimeout = 100 * time.Millisecond
 
 	panicIllegalAccessCoroutineState = "getState: illegal access from outside of workflow context"
+	unhandledUpdateWarningMessage    = "Workflow finished while update handlers are still running. This may have interrupted work that the" +
```

Comment:
Make sure that there is a space at the beginning/end of each concatenated string here or you'll get `theupdate`